configure.py

```diff
@@ -66,22 +65,30 @@ def __init__(self, qmake):
         default="",
         help="Add path to the built QOnScreenKeyboard library. This will disable building it",
     )
+    parser.add_argument(
```

Comment:
Yeah, we should consider root path option, this can be helpful for this better than adding site-packages